elrond-wasm-derive/src/parse_attr/payable_attr.rs

```diff
@@ -0,0 +1,67 @@
+use super::attr_names::*;
+use super::util::*;
+
+static EGLD_DEFAULT: &str = "EGLD";
```

Comment:
Use &[u8] instead of &str?

Reply:
This is in the macros not the contract, it is ok to work with string here.

Reply:
It might make sense to validate the token names somewhat, so as not to contain funny characters. However, this is I think more relevant when issuing tokens, here we just consume them. Having an invalid token will just cause the smart contract to indirectly not be payable. I'm not fond of having too many restrictions where there is no security vulnerability.